src/main/resources/tables.sql

```diff
@@ -0,0 +1,21 @@
+USE chess;
+
+DROP TABLE board;
+
+CREATE TABLE board
```

Comment:
```suggestion
CREATE TABLE boards
```
테이블 명은 보통 복수로 쓰는것 같아요

Reply:
boards 테이블의 필드들을보니 
사실 pieces 가 더 적절하지 않을까 라는생각이드네요

요구사항이 도메인을 변경하지 말라는 것도 있어서 변경 요청을 드리는 것은 아닙니다.
board 도메인안에서 pieces가  Map<Position, Piece> 로 관리되고 있는데 
piece에서 position을 가지고 있다면 좀 더 테이블의 명칭이 적절하지 않을까라는 생각이고
다음에 객체를 설계할때는 어떻게 접근해볼지 고민해보면 좋을것 같아서 코멘트 남겼습니다!

Reply:
동의합니다. db상으로는 pieces라는 명칭이 더 올바르다고 느껴지는 것 같아요.

position을 누가 가지느냐에 대해선 많은 고민을 해봤지만, 결국 piece가 알아야 하는건 절대적인 position이 아닌 상대적인 movement라고 생각하여, board가 갖도록 했습니다.